src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -59,11 +69,14 @@ public async Task DetectApiChanges(string buildId, string artifactName, string f
                 var pullRequestModel = await _pullRequestsRepository.GetPullRequestAsync(prNumber, repoName, filePath);
                 if (pullRequestModel == null)
                 {
+                    var repoInfo = repoName.Split("/");
```

Comment:
We should probably do some validation here instead of assuming this split correctly.

Reply:
Added validation